src/instrumentProvider.ts

```diff
@@ -197,7 +197,8 @@ export class InstrumentProvider implements VscTdp, vscode.Disposable {
             .filter(
                 (c) =>
                     c.status === ConnectionStatus.Active ||
-                    c.status === ConnectionStatus.Connected,
+                    c.status === ConnectionStatus.Connected ||
+                    c.status === ConnectionStatus.Inactive,
             )
```

Comment:
Is there another possible `ConnectionStatus`? You could probably just remove this filter if there isn't. Save a few clock cycles.